src/main/resources/messages/messages.properties

```diff
@@ -664,6 +674,7 @@ message.download_translations.preserve_hierarchy_warning=Because the @|bold 'pre
 message.language.list=@|yellow %s|@ @|green %s|@
 
 message.branch.list=@|yellow #%d|@ @|green %s|@
+message.branch.clone_success=@|green,bold '%s'|@ @|green was cloned successfully from branch |@@|green,bold '%s'|@
```

Comment:
Let's output the same response as we have for the `branch add` command - info about the created branch:

```
$ crowdin branch clone main new-branch
✔️  Fetching project info
✔️  Cloning branch (100%)
✔️  #id new-branch
```

`#id` - yellow, `new-branch` - green.